.github/workflows/codeql.yml

```
@@ -0,0 +1,50 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

The default branch in this repo is named "main".

```suggestion
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main" ]
```